docs/content/intro/quickstart.md

```diff
@@ -202,27 +202,22 @@ brig login --insecure --server https://IP_ADDRESS --root
 A Brigade [project] defines event handlers, such as the definition of a CI pipeline.
 In this example project, the handler prints a message using a string passed in the event payload.
```

Comment:
I don't believe this is an accurate description of the project that `brig init` generates.

I would probably just cut this line entirely instead of rewording it.